app/views/media_objects/_destroy_checkout.html.erb

```diff
@@ -22,7 +22,7 @@ Unless required by applicable law or agreed to in writing, software distributed
       <span class="time">00:00 <br />hh:mm</span>
     </div>
     <div class="col-5 col-sm-3 col-md-4">
-      <%= link_to 'Return now', return_checkout_url(current_checkout), class: 'btn btn-danger', method: :patch,
+      <%= link_to 'Return now', return_checkout_url(current_checkout), class: 'btn btn-primary', method: :patch,
```

Comment:
What do you think about changing this to `btn-outline`?  I think that would match the other buttons on the page more.
```suggestion
      <%= link_to 'Return now', return_checkout_url(current_checkout), class: 'btn btn-outline', method: :patch,
```

Reply:
Jon put `btn-primary` in the ticket so its what I went with, but `btn-outline` would be consistent with the button styling on the checkouts page and as you said in general matches a bit better.

Reply:
Sorry, I missed that in the ticket.  Why don't we have @joncameron weigh in before merging.

Reply:
I agree, let's go with btn-outline.